ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -16,6 +16,11 @@
 @import PSPDFKit;
 @import PSPDFKitUI;
 
+// Custom annotation toolbar subclass that adds a "Clear" button that removes all visible annotations OR the current drawing view state.
```

Comment:
This comment seems outdated, since we are not handling anything with the drawing view state.